lib/rucio/cli/bin_legacy/rucio.py

```diff
@@ -516,16 +516,16 @@ def list_scopes(args, client, logger, console, spinner):
 
     List scopes.
     """
-    if cli_config == 'rich':
+    if (cli_config == 'rich') or (not args.csv):
         spinner.update(status='Fetching scopes')
         spinner.start()
 
     if args.account:
         scopes = client.list_scopes_for_account(args.account)
     else:
         scopes = client.list_scopes()
-    if cli_config == 'rich':
-        scopes = [[scope] for scope in sorted(scopes) if 'mock' not in scope]
```

Comment:
Can you clarify on this logic change? I think it should also be separated in another commit since "[Client: Add csv arg for list-scope](https://github.com/rucio/rucio/pull/7684/commits/35b8d7c9141a87416f1a1f938c70b3751fb52365)" doesn't really reflect this change